src/firebolt_ingest/table_service.py

```diff
@@ -145,13 +145,22 @@ def insert_full_overwrite(
         # recreate the table
         cursor.execute(query=internal_table_schema)
 
+        def is_column_present(column, internal_table_columns):
+            name, type_ = column.name, column.type
+            return any(
+                [
+                    (name, type_) in internal_table_columns,
+                    (name, type_.replace("NTZ", "")) in internal_table_columns,
+                ]
+            )
```

Comment:
While this seems clean, we only have one case where NTZ is present - TIMESTAMP NTZ. IMO it will be clearer to do a simple "if" and maybe pop a comment that NTZ can sometimes be represented by an alias internally.

Reply:
Yeah, I get the idea why you're adding it. I just mean that there's a single case where this is applicable - TIMESTAMPNTZ. Seems a bit redundant to do this check twice for each column we encounter, since we're looping over each column in a table.
What I suggest is something like,
```python
name, type_ = column.name, column.type
present = (name, type_) in internal_table_columns
# TIMESTAMPNTZ is sometimes represented as TIMESTAMP, need to check both
if type_ == "TIMESTAMPNTZ":
  present = present || (name, "TIMESTAMP") in internal_table_columns
return present
```
This might not be necessarily as pretty or pythonic as the one above, but the intent here is much clearer.

Reply:
Done, also added integration test for TIMESTAMPTZ